blocks/media-gallery/media-gallery.js

```diff
@@ -0,0 +1,32 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+
+export default function decorate(block) {
+    // modify card structure to have a card-content div
+    for(let idx = 0; idx < block.children.length; ++idx) {
+        const child = block.children[idx];
+        child.classList.add('card');
+        child.classList.add(`card-${idx}`);
+
+        const cardContent = document.createElement('div');
+        cardContent.classList.add('card-content');
+
+        console.log(block.classList);
+        if(block.classList.contains('alternate-2-1')) {
+            const startButton = document.createElement('button');
+            startButton.classList.add('start-button');
+            startButton.innerHTML = "<svg aria-hidden='true' class='svg' fill='currentColor' height='100%' viewBox='0 0 24 24' width='100%' xmlns='http://www.w3.org/2000/svg'><path d='M6 1.773l15 10.23L6 22.226z'></path></svg>";
```

Comment:
As they're using a "calcite-button", can you use that directly instead of a custom implementation? Their calcite web components should be available to use within the project already.
```
<calcite-button appearance="solid" kind="inverse" color="light" scale="l" round="" icon-start="play-f" dir="ltr" alignment="center" width="auto" label="play video - How does the US count everyone?" calcite-hydrated="" type="button">
	</calcite-button>
```

Reply:
I'd also suggest using dom-helpers to add the components and adding calcite component helpers for it as needed (calciteButton, calciteCard, ...).
https://github.com/andreituicu/franklin-jsx-like/blob/main/scripts/dom-helpers.js

This is script is becoming a standard in most Edge Delivery projects.